channeldb/channel.go

```diff
@@ -827,6 +839,42 @@ func fetchChanBucketRw(tx kvdb.RwTx, nodeKey *btcec.PublicKey, // nolint:interfa
 // fullSync syncs the contents of an OpenChannel while re-using an existing
 // database transaction.
 func (c *OpenChannel) fullSync(tx kvdb.RwTx) error {
+	// Fetch the outpoint bucket and check if the outpoint already exists.
+	opBucket := tx.ReadWriteBucket(outpointBucket)
+	if opBucket == nil {
```

Comment:
same here, no need to check

Reply:
This is what I thought but one of the itests keeps failing on this check.

Reply:
this is an impossible case to reach